### PR TITLE
Add HW 5-6

### PR DESCRIPTION
**Pull request by 1stnik**

### CODE REVIEW

File: TaskManagementSystem/src/main/java/com/hillel/task_management_system/controller/TaskController.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class TaskController {
+
+    private final TaskService taskService;
+
+    private final UserService userService;
```
Comment (kartius) [inline TaskManagementSystem/src/main/java/com/hillel/task_management_system/controller/TaskController.java:17]: Why did you autowire UserSrvice into TaskController?

File: TaskManagementSystem/src/main/java/com/hillel/task_management_system/TaskManagementSystemApplication.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+import com.hillel.task_management_system.service.UserService;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+
+@SpringBootApplication
+public class TaskManagementSystemApplication {
+
+    public static void main(String[] args) {
+        ConfigurableApplicationContext applicationContext =
```
Comment (kartius) [inline TaskManagementSystem/src/main/java/com/hillel/task_management_system/TaskManagementSystemApplication.java:27]: Now you shouldn't communicate with services via context beans. Now the communication is going through the API

Comment (1stnik) [reply TaskManagementSystem/src/main/java/com/hillel/task_management_system/TaskManagementSystemApplication.java:27]: Of course, but I want to see what it is going on in the console too

File: TaskManagementSystem/src/main/java/com/hillel/task_management_system/controller/TaskController.java (hunk 1 above)
Comment (1stnik) [reply TaskManagementSystem/src/main/java/com/hillel/task_management_system/controller/TaskController.java:17]: Sorry, I didn't get what you mean

Comment (1stnik) [reply TaskManagementSystem/src/main/java/com/hillel/task_management_system/controller/TaskController.java:17]: I've used userService in a couple of methods of taskService: showUserTasks and deleteUserFromListOfTask.

**Pull request merged**